cmd/mirror-main.go

```diff
@@ -587,6 +591,9 @@ func (mj *mirrorJob) mirror(ctx context.Context, cancelMirror context.CancelFunc
 			}
 		}
 		if err != nil {
+			if err.ToGoError().Error() == ErrBucketDoNotExist.Error() {
```

Comment:
This is not correct style expected style is to avoid string matching instead rely on types.

```
switch err.ToGoError().(type) {
case BucketDoesNotExist:
```

Reply:
Yes will change it to a struct with a error as anis suggested